core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -185,8 +189,12 @@ package object config {
     .createWithDefaultString("1g")
 
   private[spark] val EXECUTOR_MEMORY_OVERHEAD = ConfigBuilder("spark.executor.memoryOverhead")
-    .doc("The amount of off-heap memory to be allocated per executor in cluster mode, " +
-      "in MiB unless otherwise specified.")
+    .doc("Amount of non-heap memory to be allocated per executor process in cluster mode " +
```

Comment:
I think that generally we reserve extended comments for the docs below. You don't need to change this.

Reply:
OK. I revert comment in core, but reserve a little change.